tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -2108,7 +2108,7 @@ TEST_F(EnergyPlusFixture, FanCoil_ASHRAE90VariableFan)
     StdRhoAir = 1.2;
 
     BeginEnvrnFlag = true;
-    InitFanCoilUnits(FanCoilNum, ZoneNum);
+    InitFanCoilUnits(FanCoilNum, ZoneNum, ZoneNum);
```

Comment:
What the heck is this? Oh, it's a unit test. InitFanCoilUnits now has arguments for FanCoilNum, ZoneNum, and ControlledZoneNum (to pass to sizing). So this change simply used ZoneNum as both arguments.